packages/adapter/src/utils.ts

```diff
@@ -175,3 +175,15 @@ export function getDecimalBN(decimals: number | string) {
   const power = new BN(decimals, 10);
   return base.pow(power);
 }
+
+/**
+ * Determine the account type based on the address.
+ * Because 1 byte takes 2 characters in hex string, we can use the length of the address to determine the address type.
+ * AccountKey20: 42 characters with prefix 0x.
```

Comment:
Let’s add the 0x check to this function.

const length = _.startWith(address, "0x")? 42: 40;

Reply:
Fixed